docs/en/migration.md

```diff
@@ -125,6 +125,21 @@ model = dict(
     init_cfg=...)
 ```
 
+**NOTE:** `data_preprocessor` can be defined outside the model dict, which has higher priority than it in model dict. For example:
```

Comment:
What is higher priority?

Reply:
[Runner](https://github.com/open-mmlab/mmengine/blob/main/mmengine/runner/runner.py#L450) can get key `data_preprocessor` in cfg directly and merge it to [cfg.model](https://github.com/open-mmlab/mmengine/blob/main/mmengine/runner/runner.py#L401)